src/logic/calculate.js

```diff
@@ -0,0 +1,8 @@
+// eslint-disable-next-line consistent-return
+const calculate = ({ total, next, operation }, buttonName) => {
+  if (buttonName === '+/-') {
+    return { total: total * -1, next: next * -1, operation };
+  }
```

Comment:
- [ ] We need to improve the calculate function. Here we need to deal with any possible button that it's pressed within the calculator. Like: `AC`, `.`, `=`, and any digits.

- [ ] Here we also need to deal with the operation. Meaning that, if it is time to do an operation, we need to call the operation function to deal with it.